src/node_os.cc

```diff
@@ -120,15 +120,15 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
   // The array is in the format
   // [model, speed, (5 entries of cpu_times), model2, speed2, ...]
   std::vector<Local<Value>> result(count * 7);
-  for (int i = 0; i < count; i++) {
+  for (int i = 0, j = 0; i < count; i++, j = i * 7) {
```

Comment:
```suggestion
  for (int i = 0, j = 0; i < count; i++) {
```

There's no need to set `j = i * 7` every loop iteration as long as there are seven `j++` in the loop body.

Reply:
@richardlau - yes, thanks! amended accordingly, ptal.